paasta_tools/chronos_tools.py

```diff
@@ -394,6 +395,15 @@ def get_healthcheck_mode(self, _):
         # Healthchecks are not supported yet in chronos
         return None
 
+    def get_desired_state_human(self):
```

Comment:
This is not actually true. These are class functions that for the serviceinstanceconfig.

A caller doesn't need to "know" anything. (notice that this patch required no changes to the client code, they were calling `get_desired_state_human` just as before) So they just run `service_config.get_desired_state_human` like normal.

All I'm doing is taking a class function that was too generic in serviceinstanceconfig and adding framework-specific versions that are more specific. (and hopefully more human readable of course)